src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div>
+    <div>
+      <label>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></label>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-owns="id"
+      :aria-label="selectLabel"
```

Comment:
Since the aria-label here is just the text instead of referring to the content of the label, "Required" doesn't get read. We can add aria-required or use aria-labelledby and add an id to the label.